src/main/java/org/folio/circulation/support/items/ItemBatchUtils.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.circulation.support.items;
+
+import org.folio.circulation.support.Result;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public final class ItemBatchUtils {
+
+  /**
+   * The optimal number of identifiers that will not exceed the permissible length
+   * of the URI in according to the RFC 2616
+   */
+  private static final int BATCH_SIZE = 40;
+
+  private ItemBatchUtils(){
+
+  }
+
+  public static CompletableFuture<Result<List<List<String>>>> mapItemIdsInBatchItemIds(List<String> itemIds) {
```

Comment:
If this is used to make multiple HTTP requests each with a subset of IDs, I believe the `MultipleRecordsFetcher` class already does this. Was that implementation inappropriate, or were you unaware of it?

Reply:
I was unaware of it. Thank you for the comment. Fixed